### ISSUE TITLE
add optional individual mz and RT columns in MzTabExporter output

### ISSUE THREAD

**Issue by lars20070** [OpenMS/OpenMS#3705]
In addition to standard `mzTab`, the `MzTabExporter` tool writes a number of very useful optional `opt_global_*` columns. Hopefully, some of them will become part of the `mzTab` standard in the future.

Individual peptide intensities are reported as `peptide_abundance_study_variable[1-n]`. Could we also report individual `mz` and `RT` values as `opt_global_*` columns in the `PEP` section [here](https://github.com/OpenMS/OpenMS/blob/develop/src/topp/MzTabExporter.cpp#L888)?

```
opt_global_mass_to_charge_study_variable[1-n]
opt_global_retention_time_study_variable[1-n]
```

One possible use case are `mzTab` based [QC reports](https://github.com/ProteomicsScripts/ProteomicsScripts/tree/master/mzTab2report). For example, in SILAC experiments one expects the heavy-labelled peptide to elude slightly earlier than its light partner. The

```opt_global_retention_time_study_variable[1] - opt_global_retention_time_study_variable[2]```

distribution should peak at negative retention times and would be a helpful QC plot.

**Comment by jpfeuffer** [comment]
I think that would make sense as optional additional columns (opt_). But keep in mind that study_variable does not necessarily correspond to exactly one ms input file.

**Comment by lars20070** [comment]
Resolved in #4047.
